packages/vendure-plugin-customer-managed-groups/src/api-extensions.ts

```diff
@@ -113,6 +133,16 @@ export class CustomerManagedGroupsResolver {
     return this.service.createCustomerManagedGroup(ctx);
   }
 
+  @Mutation()
+  @Allow(Permission.Authenticated)
+  async makeCustomerAdminOfGroup(
```

Comment:
Was this needed for this pr? It would be nice if each feature gets its own pr. For now its ok.

Can we rename to makeCustomerAdminOfCustomerManagedGroup? Its a bit long, but prevents graphql conflicts with possible future mutations

Reply:
The last test case involves an admin updating profile of another admin of the same group. I think we might need this to assign another admin to an existing customer managed group